README.md

```diff
@@ -0,0 +1,56 @@
+# vpphelper
+
+vpphelper provides helper functions (using govpp) that helps to connect to VPP and configure VPP using a configuration (.conf) file.
+
+## StartAndDialContext
+`StartAndDialContext` function starts VPP using a VPP configuration file and returns a `Connection` to VPP. 
+
+```go
+conn, vppErrCh := vpphelper.StartAndDialContext(connectCtx)
+```
+```go
+func StartAndDialContext(ctx context.Context, opts ...Option) (conn Connection, errCh <-chan error) {
+	o := &option{
+		rootDir:   DefaultRootDir,
+		vppConfig: vppConfContents,
+	}
+	for _, opt := range opts {
+		opt(o)
+	}
+
+	if err := writeDefaultConfigFiles(ctx, o); err != nil {
+		errCh := make(chan error, 1)
+		errCh <- err
+		close(errCh)
+		return nil, errCh
+	}
+	logWriter := log.Entry(ctx).WithField("cmd", "vpp").Writer()
+	vppErrCh := exechelper.Start("vpp -c "+filepath.Join(o.rootDir, vppConfFilename),
+		exechelper.WithContext(ctx),
+		exechelper.WithStdout(logWriter),
+		exechelper.WithStderr(logWriter),
+	)
+	select {
+	case err := <-vppErrCh:
+		errCh := make(chan error, 1)
+		errCh <- err
+		close(errCh)
+		return nil, errCh
+	default:
+	}
+
+	return DialContext(ctx, filepath.Join(o.rootDir, "/var/run/vpp/api.sock")), vppErrCh
+}
```

Comment:
What was the logic of including the full code for StartAndDialContext here?

Reply:
I thought that it will be easy to understand the explanations mentioned below full code, if the full code is visible right above itself.

I can make changes as you suggest.

Reply:
:)  I am delighted you found the code clear and understandable... usually folks read the readme though for a simple 'this is how you do x' kinds of stuff :)